### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by ahgnaw**
Use sphinx to generate documentation pages in a separate directory 'empyrical-docs'.

### CODE REVIEW

**Comment by abhijeetkalyan** [comment]
I don't know enough about Sphinx to review this - @richafrank or @llllllllll might be better suited here?

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+empyrical
+==============
+Common financial risk metrics.
+
+Installation
+-----------------
+::
```
Comment (llllllllll) [inline README.rst]: `.. code-block:: bash`

File: docs/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.. empyrical documentation master file, created by
```
Comment (llllllllll) [inline docs/source/index.rst]: why not include the readme in the index instead of its own page?

File: empyrical/stats.py [hunk 3]
```diff
@@ -47,22 +47,18 @@ def annualization_factor(period, annualization):
 
     Parameters
     ----------
-    period : str, optional
+    :param period:
```
Comment (llllllllll) [inline empyrical/stats.py]: why did you change the docstrings from numpy style?

**Comment by llllllllll** [comment]
can we use the readthedocs theme, we use it for zipline and I think it looks nicer than the default

**Comment by llllllllll** [comment]
we also need intersphinx links for third party libs, I would look at the sphinx conf for zipline

**Comment by llllllllll** [comment]
#12

**Comment by ahgnaw** [comment]
@llllllllll Sure, i have no theme preferences.

I've gone ahead an added intersphinx links, changed the readme

I changed the docstring style because the docstrings were not compiling correctly. The documents produced looked wonky.

**Comment by llllllllll** [comment]
Why is the build output dir outside of the repo, it is normally just `docs/build`

**Comment by twiecki** [comment]
Is this PR still active? Should we merge it?

**Comment by rsheftel** [comment]
Is this PR active? I have setup sphinx documentation on my fork that creates .rst files that could be used to automatically update the documentation if we hosted on ReadTheDocs. I am happy to create a PR for that if setting up ReadTheDocs something the package maintainers would like to do. Let me know.

**Comment by twiecki** [comment]
that'd be great!

**Comment by rsheftel** [comment]
New PR #75 opened to replace this PR

**Comment by rsheftel** [comment]
Now that #75 has been merged this PR is not needed and can be closed
